### PR TITLE
Add switch region location feature

### PR DESCRIPTION
**Pull request by oleksandr-holovatyi**

### CODE REVIEW

File: ui/src/test/java/core/SelenideProvider.java [hunk 1]
```diff
@@ -32,7 +32,8 @@
     public static void init() {
         Configuration.baseUrl = "http://";
         Configuration.browser = Browsers.CHROME;
-        Configuration.browserSize = "1920x1080";
+//        Configuration.browserSize = "1920x1080";
```
Comment (olegasko) [inline ui/src/test/java/core/SelenideProvider.java]: try do not leave commented code

File: ui/src/test/java/stepdefinitions/StepDefinitions.java [hunk 2]
```diff
@@ -22,10 +22,34 @@ public void whenExample() {
         page.openLocationMenu();
     }
 
-    @Then("menu is open")
-    public void thenExample() {
+//    @Then("menu is open")
```
Comment (olegasko) [inline ui/src/test/java/stepdefinitions/StepDefinitions.java]: try do not leave commented code

File: ui/src/test/java/stepdefinitions/StepDefinitions.java [hunk 3]
```diff
@@ -22,10 +22,34 @@ public void whenExample() {
...
+//    public void thenExample() {
+//        assertThat(page.isMenuVisible())
+//                .as("Menu is not visible")
+//                .isTrue();
+//    }
+
+    @Then("blbla")
```
Comment (olegasko) [inline ui/src/test/java/stepdefinitions/StepDefinitions.java]: remove

File: ui/src/test/java/stepdefinitions/StepDefinitions.java [hunk 4]
```diff
@@ -22,10 +22,34 @@ public void whenExample() {
...
+    @When("menu is open")
```
Comment (olegasko) [inline ui/src/test/java/stepdefinitions/StepDefinitions.java]: When shouldn't contain assertion

File: ui/src/test/java/stepdefinitions/StepDefinitions.java [hunk 5]
```diff
@@ -22,10 +22,34 @@ public void whenExample() {
...
+    public void whenMenuIsOpen() {
         assertThat(page.isMenuVisible())
                 .as("Menu is not visible")
                 .isTrue();
     }
+
+    @Then("activate Location 'Global (EN)'")
```
Comment (olegasko) [inline ui/src/test/java/stepdefinitions/StepDefinitions.java:34]: assertion should be in separate 'then' step

File: ui/src/test/java/stepdefinitions/StepDefinitions.java [hunk 6]
```diff
@@ -22,10 +22,34 @@ public void whenExample() {
...
+    public void activateLocationGlobalEN() {
+        page.activateLocationGlobalEN();
+        assertThat(page.isLocationGlobalEN())
+                .as("Location is not Global (EN)")
+                .isTrue();
+    }
+
+    @Then("activate Location 'Україна (UA)'")
```
Comment (olegasko) [inline ui/src/test/java/stepdefinitions/StepDefinitions.java:42]: assertion should be in separate 'then' step

File: ui/src/test/resources/practice1.feature [hunk 7]
```diff
@@ -0,0 +1,16 @@
+@tmsLink=EQFX-POC-02
+Feature: practice1 feature
```
Comment (olegasko) [inline ui/src/test/resources/practice1.feature]: named it properly

File: ui/src/test/resources/practice1.feature [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+
+  Scenario: Base practice scenario4
+    Given on the EPAM main page
+    When click on Menu button
+    When menu is open
+    Then activate Location 'Global EN'
+
+
+  Scenario: Base practice scenario5
```
Comment (olegasko) [inline ui/src/test/resources/practice1.feature]: named it properly

File: ui/src/test/java/pages/MainPage.java [hunk 9]
```diff
@@ -2,13 +2,18 @@
...
+import org.openqa.selenium.By;
 
 import static com.codeborne.selenide.Selenide.$;
 
+//TODO refactor extend main page, move not common things in separate classes
 public class MainPage {
 
     SelenideElement locationButton = $(".location-selector__button");
     SelenideElement locationsList = $(".location-selector__list");
+    SelenideElement LocationGlobalENButton = $(By.xpath(".//ul[@class='location-selector__list']//a[text()='Global ']"));
```
Comment (olegasko) [inline ui/src/test/java/pages/MainPage.java]: $x(".//ul[@class='location-selector__list']//a[text()='Global ']") - $x means find by xpath in Selenide

File: ui/src/test/java/pages/MainPage.java [hunk 10]
```diff
@@ -2,13 +2,18 @@
...
+    SelenideElement LocationUkraineUAButton = $(By.xpath(".//ul[@class='location-selector__list']//a[text()='Україна ']"));
```
Comment (olegasko) [inline ui/src/test/java/pages/MainPage.java]: change using pattern above

**Review by oleksandr-holovatyi — COMMENTED** [review]
Review resolved

File: ui/src/test/java/stepdefinitions/StepDefinitions.java (hunk 6 above)
Comment (oleksandr-holovatyi) [reply ui/src/test/java/stepdefinitions/StepDefinitions.java:42]: Added to TODO

**Review by oleksandr-holovatyi — COMMENTED** [review]
Review submitted. All done, exsept 1 issue added to TODO

File: ui/src/test/java/stepdefinitions/StepDefinitions.java [hunk 11]
```diff
@@ -23,9 +24,26 @@ public void whenExample() {
     }
 
     @Then("menu is open")
-    public void thenExample() {
+    public void whenMenuIsOpen() {
```
Comment (olegasko) [inline ui/src/test/java/stepdefinitions/StepDefinitions.java:27]: thanMenuIsOpen()

**Review by olegasko — APPROVED** [review]

**Pull request merged**